### PR TITLE
feat: EU scrapers round 2

### PR DESCRIPTION
**Pull request by qgolsteyn**
This PR adds support for the following countries:

- Ireland (gov source, number of cases regionally)
- Austria (gov source, backfilled with community project, number of cases regionally)
- Poland (gov source, backfilled with community project, number of cases and deaths regionally)
- Latvia (gov source, number of cases regionally)
- Lithuania (gov source, number of cases regionally)
- Sweden (gov source, number of cases regionally, additional country-wide data)
- Slovenia (gov source, number of cases regionally, additional country-wide data)

All scraper uses ISO codes. @hyperknot be aware I have missing population for Slovenia, Ireland, Poland, and Lithuania.

### CODE REVIEW

**Comment by qgolsteyn** [comment]
image

**Comment by qgolsteyn** [comment]
image

**Comment by qgolsteyn** [comment]
image

**Comment by qgolsteyn** [comment]
image

**Comment by qgolsteyn** [comment]
image

**Comment by qgolsteyn** [comment]
image

**Comment by qgolsteyn** [comment]
image

**Comment by lazd** [comment]
Awesome work!

File: src/shared/scrapers/PL/index.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        });
+      }
+
+      if (data.length > 0) data.push(transform.sumData(data));
+
+      return data;
+    },
+    '2020-04-13': async function() {
+      this.url = 'https://www.gov.pl/web/koronawirus/wykaz-zarazen-koronawirusem-sars-cov-2';
+      this.sources = [
```
Comment (ciscorucinski) [inline src/shared/scrapers/PL/index.js:55]: Incorrect source
```
this.sources = [
    {
      url: 'http://www.mz.gov.pl/',
      name: 'Ministry of Health of the Republic of Poland'
    }
  ]
```

Comment (qgolsteyn) [reply src/shared/scrapers/PL/index.js:55]: Thank you, I will get this in

**Comment by hyperknot** [comment]
Thanks, this is really great!

**Pull request merged**